packages/server/src/routes/videos/videos.ts

```
@@ -0,0 +1,27 @@
+import { createRoute, RouteHandler, z } from "@hono/zod-openapi";
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)